Infrastructure.Services/Http/EndpointValidationService.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Infrastructure.Services.Extensions;
+using Polly;
+using Serilog;
+
+namespace Infrastructure.Services.Http
+{
+    public class EndpointValidationService : IEndpointValidationService
+    {
+        private const string ChromeUserAgent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.36";
```

Comment:
Er denne ikke skrøbelig? Måske der skal være noget note et sted hvis den pludselig ender med at give mange fejl pga en "død" useragent

Reply:
Det handler om identitet. Jeg tror ikke at webserverere pludselig afviser et http request fordi agenten er for "gammel" det er typisk front-end app der brækker, men jo det er jo et øjebliksbillede. Uden den bliver vi faktisk afvist af strongminds.dk :-)